src/utils/AFramePhysics.js

```diff
@@ -80,6 +81,50 @@ AFRAME.registerComponent("shadowcustomsetting", {
     },
 });
 
+AFRAME.registerComponent("materialinfo", {
```

Comment:
this should probably be placed in a separate file as it's not part of the a-frame physics system. (see https://github.com/engaging-computing/MYR/blob/master/src/utils/AframeRegIndicator.js for example)